app/models/mixins/aggregation_mixin.rb

```diff
@@ -1,23 +1,22 @@
 module AggregationMixin
   extend ActiveSupport::Concern
   included do
-    virtual_column :aggregate_cpu_speed,       :type => :integer, :uses => :hosts
-    virtual_column :aggregate_cpu_total_cores, :type => :integer, :uses => :hosts
-    virtual_column :aggregate_physical_cpus,   :type => :integer, :uses => :hosts
-    virtual_column :aggregate_memory,          :type => :integer, :uses => :hosts
-    virtual_column :aggregate_vm_cpus,         :type => :integer, :uses => :vms_and_templates
-    virtual_column :aggregate_vm_memory,       :type => :integer, :uses => :vms_and_templates
-    virtual_column :aggregate_disk_capacity,   :type => :integer, :uses => :hosts
+    virtual_aggregate :aggregate_vm_memory, :vm_hardwares, :sum, :memory_mb
+    virtual_aggregate :aggregate_vm_cpus, :vm_hardwares, :sum, :cpu_sockets
+    virtual_aggregate :aggregate_physical_cpus, :host_hardwares, :sum, :cpu_sockets
+    virtual_aggregate :aggregate_disk_capacity, :host_hardwares, :sum, :disk_capacity
+    virtual_aggregate :aggregate_memory, :host_hardwares, :sum, :memory_mb
+    virtual_aggregate :aggregate_cpu_total_cores, :host_hardwares, :sum, :cpu_total_cores
+    virtual_aggregate :aggregate_cpu_speed, :host_hardwares, :sum, :aggregate_cpu_speed
 
-    alias_method :all_hosts,              :hosts
-    alias_method :all_host_ids,           :host_ids
-    alias_method :all_vms_and_templates,  :vms_and_templates
-    alias_method :all_vm_or_template_ids, :vm_or_template_ids
-    alias_method :all_vms,                :vms
-    alias_method :all_vm_ids,             :vm_ids
-    alias_method :all_miq_templates,      :miq_templates
-    alias_method :all_miq_template_ids,   :miq_template_ids
```

Comment:
So my assumption here is that "all" of the "`all_*` methods" were only used by `ems_cluster.rb` (and were written by Smash Mouth... #badJoke), and as such, moved up to that class.  Correct?

Reply:
yes

Reply:
there used to be a lot of case statements. I split this up into 2 different files, that were similar but had different 

think nick deleted the second file though.

Reply:
Some "`git` proof" of this statement would be nice.

Reply:
I was optimistic and unfortunately nick did not delete that file.

https://github.com/ManageIq/manageiq/blob/master/app/models/mixins/relationships_aggregation_mixin.rb

Reply:
> there used to be a lot of case statements. I split this up into 2 different files, that were similar but had different

This is what I wanted the "git proof" for (a commit/PR/etc. of this happening), and not a link.

> yes, it's unfortunate and mildly disgusting, but here we are

Assuming this is answering "yes" to my original and poorly worded question, I think it is safe to resolve this.